pkg/skaffold/build/ko/build_test.go

```diff
@@ -0,0 +1,129 @@
+/*
+Copyright 2021 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ko
+
+import (
+	"bytes"
+	"context"
+	"strings"
+	"testing"
+
+	"github.com/docker/docker/client"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/google/ko/pkg/build"
+	"github.com/google/ko/pkg/publish"
+
+	// latestV1 "github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest/v1"
+	latestV1 "github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/ko/schema"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/docker"
+	"github.com/GoogleContainerTools/skaffold/testutil"
+)
+
+// koImportPath is the import path of this package, with the ko scheme prefix.
+const koImportPath = "ko://github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/ko"
```

Comment:
Aside: we seem to have broken this `ko://` behaviour (#5505).

Reply:
I can take a look at this - feel free to assign me (I don't have permission to assign myself).

Reply:
Fixed in #6286